canonical-websites/www.ubuntu.com
# Flattened /legal/dataprivacy

## Done

- Flattened legal data-privacy section
- Updated navigation
- Updated redirects
- Added some special redirects requested by legal team

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- look at the files in the /legal directory and compare to this [spreadsheet](https://docs.google.com/spreadsheets/d/12kuuGgwRqN3k7wlRc4QKIuZifjvxbynizO_LEjKjBq4/edit#gid=0)
* test the link [/legal/data-privacy](http://0.0.0.0:8001/legal/data-privacy) and [the old link](http://0.0.0.0:8001/legal/dataprivacy) redirects also test that [/data-privacy](http://0.0.0.0:8001/data-privacy) redirects as well


## Issue / Card

Fixes #4194
